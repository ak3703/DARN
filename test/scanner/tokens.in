()
[
]
{
}
;
,
:
+
-
*
/
=
==
!=
<
<=
>
>=
&&
||
!
if
else
elif
for
while
return
true
false
char
int
float
bool
2
2.0
hello
'a'
"this is a string"
#
&