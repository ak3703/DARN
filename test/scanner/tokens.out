LPAREN
RPAREN
LBRACK
RBRACK
LCURLY
RCURLY
SEMI
COMMA
COLON
PLUS
MINUS
TIMES
DIVIDE
ASSIGN
EQ
NEQ
LT
LEQ
GT
GEQ
AND
OR
NOT
IF
ELSE
ELIF
FOR
WHILE
RETURN
TRUE
FALSE
CHAR
INT
FLOAT
BOOL
INTLITERAL
FLOATLITERAL
ID
CHARLITERAL
STRINGLITERAL
OCTOTHORP
PERCENT
